relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -340,6 +382,57 @@ pub(crate) fn extract_tags(span: &Span, config: &Config) -> BTreeMap<SpanTagKey,
             let truncated = truncate_string(scrubbed_desc.to_owned(), config.max_tag_value_size);
             span_tags.insert(SpanTagKey::Description, truncated);
         }
+
+        if span_op.starts_with("resource.") {
+            if let Some(http_response_content_length) = span
+                .data
+                .value()
+                .and_then(|data| data.get("http.response_content_length"))
+                .and_then(|value| value.as_str())
+            {
+                span_tags.insert(
+                    SpanTagKey::HttpResponseContentLength,
+                    http_response_content_length.to_owned(),
```

Comment:
@phacops this is something we did not have to deal with before: We want to propagate numeric attributes to snuba, but `sentry_tags` can only contain string values. Would you prefer we leave these values in `span.data` instead (and letting the sentry-side consumer transfer them to snuba as numbers somehow)?